config/jobs/kubernetes-sigs/promo-tools/promo-tools.yaml

```diff
@@ -200,3 +200,39 @@ presubmits:
       testgrid-dashboards: sig-release-releng-presubmits
       testgrid-num-columns-recent: '30'
       testgrid-alert-email: release-managers+alerts@kubernetes.io
+  - name: pull-cip-one-time-run
+    decorate: true
+    path_alias: "sigs.k8s.io/promo-tools"
+    skip_report: false
+    always_run: false
+    labels:
+      preset-dind-enabled: "true"
```

Comment:
/hold

can we not introduce new jobs that will run in the default cluster that is no longer available next month?

/cc @xmudrii (after your holidays)

Reply:
We're not proceeding with this job at the moment, it'll anyways require some changes, so we can ignore it for now.